src/tracing/trace_event.h

```diff
@@ -0,0 +1,1710 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SRC_TRACING_TRACE_EVENT_H_
+#define SRC_TRACING_TRACE_EVENT_H_
+
+#include <stddef.h>
+
+#include "v8-platform.h"
```

Comment:
Oh, the macro uses didn't come up in my search in VS because `src/tracing/trace_event.h` isn't referenced by `node.vcxproj` generated by the build. That header needs to be added to [`node.gyp` here](https://github.com/nodejs/node/pull/9304/files#diff-259e627be3fafed9e9c10a7fcb26879eR186).